plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	defaultMaxDatumsPerCall        = 20     // PutMetricData only supports up to 20 data metrics per call by default
+	defaultMaxDatumsPerCall        = 1000   // PutMetricData only supports up to 1000 data metrics per call by default
```

Comment:
No blocker from me: Are there any public documents regarding the PMD Metric Datum that we could put here for future reference? Another thing would be [PutMetricData documents](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricData.html) only show 20 MetricDatum and 10 dimensions  instead of 1000 MetricDatum and 30 dimensions. This brings me the contrast of referencing public documents when comparing to CloudWatchAgent default PMD calls.